src/main/java/com/kodar/academy/Library/model/dto/book/BookCreateDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.kodar.academy.Library.model.dto.book;
+
+import com.kodar.academy.Library.model.dto.author.AuthorDTO;
+import com.kodar.academy.Library.model.dto.genre.GenreDTO;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+public class BookCreateDTO {
+
+    private String isbn;
+    private String title;
+    private String publisher;
+    private short year;
+    private LocalDateTime dateAdded;
```

Comment:
Date added shouldn't be sent with the request but instead set during entity creation. According to above comment you can leave dateAdded here and remove it from BookCreateRequest